nixos/modules/services/web-apps/guacamole.nix

```diff
@@ -0,0 +1,138 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.guacamole;
+  uid = 54201;
+  gid = 54201;
```

Comment:
Rather than sticking with the arbitrarily-chosen free IDs I was using, you'll need to append a guacamole user and group to the ID lists in `<nixpkgs/nixos/modules/misc/ids.nix>`, then refer to those here like `config.ids.uids.guacamole`. This prevents UID/GID collisions on NixOS systems.

Reply:
Indeed, should do.

Reply:
This is actually not needed, NixOS will automatically select a free uid/gid if you don't give it one at all